### ISSUE TITLE
deselect the zoom to area option

### ISSUE THREAD

**Issue by fazzer14** [GEOLYTIX/xyz#250]
Seems like as soon as click the zoom to area option I can't get back to the normal select tool. Is this user error?

**Comment by dbauszus-glx** [comment]
No. Interactions are not handled well. The Zoom to area like any other interaction (e.g. edit) must finish the current interaction (e.g. select). If an interaction is finished the default interaction (select) should be activated. This must happen from the interaction itself. Any interaction which is activated should automatically finish the current interaction. Any interaction other than the select interaction which is finished by itself should reactivate the select interaction.

Button states are not handled well. We need to reevaluate a smart way to display button activity.
